All my updates

evandrocoan opened the pull request.

evandrocoan committed:
Added to ignore the current view if it is an AMXX file, when the

package amxmodx is enabled. This is because the amxmodx package is
already generating auto completions for the current view.

evandrocoan committed:
Created new settings for the packages amxmodx and All Autocomplete:

Enable auto completing words from the current file.
"word_autocomplete": true,

Use the package `All Autocomplete` to do the completing on the
current file.

evandrocoan committed:
Moved the amxmodx package handling to there and only disabled this

package when the amxmodx package is enabled.

evandrocoan committed:
Update from upstream repo alienhard/SublimeAllAutocomplete

# Conflicts:
#	all_views_completions.py

evandrocoan committed:
Merge branch 'pulls/1'

evandrocoan committed:
Moved the All Autocomplete from Amxmodx to All Autocomplete package

fixes evandrocoan/ITE#86

evandrocoan committed:
Created the missing dependencies.json for the PythonDebugTools

evandrocoan committed:
Added to Debugger class to extend loggging.Logger and deprecated

the direct instantiation of the Debugger class by the getLogger()
factory.

evandrocoan committed:
Renamed python_debug_tools to debug_tools

alienhard commented:
Hi, thanks for sending the pull request. Can you explain what exactly the change does? Is the goal to just disable the plugin for AMXX files?

evandrocoan committed:
Cleaned up dead code.

evandrocoan commented:
They where related to AMXX on the beginning, but now I removed all that stuff into the Amxx own package. The only thing for AMXX there is currently is the debug comments I had forgot to rename.

The changes are:

1. Added a logger module "PythonDebugTools"
1. Removed the `if is_disabled_in(view.scope_name(locations[0])):`, oops. I should add this line back.
1. Created the plugin_unloaded() to remove the on_settings change hook when the plugin is unloaded by Sublime Text.
1. More empty lines 
1. Removed MAX_WORDS_PER_VIEW with without_duplicates() because it has complexity O(n^2)

evandrocoan committed:
Fixed UnboundLocalError: local variable 'view' referenced before

assignment.

evandrocoan committed:
Renamed the PythonDebugTools dependency to debug_tools

evandrocoan committed:
Renamed debug_tools dependency to DebugTools

evandrocoan committed:
Added new dependencies for DebugTools on dependencies.json

evandrocoan committed:
Removed unnecessary view_words = None

evandrocoan committed:
Fixed dependencies not being installed on windows, PC only installs

the dependencies in the most specific and matching list.
Issue looking for a fix for this with a new system:
Ideas for dependency specification
https://github.com/wbond/package_control/issues/819

evandrocoan committed:
Implemented all word completion for all windows, instead of for the

current window only.

evandrocoan committed:
Fixed duplicated words being added to the auto completion list

evandrocoan committed:
Created a .no-sublime-package for all packages, in attempt to fix

evandrocoan/ITE 108 - Expand Region keybind is not working on stable version

evandrocoan committed:
Revert "Created a .no-sublime-package for all packages, in attempt to fix"

This reverts commit 4d15972414ecc724af221a2b006d5e355fec0a83.

evandrocoan committed:
Renamed the dependency DebugTools to lowercase name

evandrocoan committed:
Renamed the dependency PortalockerFiles to lowercase name

evandrocoan committed:
Renamed the dependency ConcurrentLogHandler to lowercase name

evandrocoan committed:
Renamed the portalockerfiles dependency to portalockerfile
